snapshot/policy/scheduling_policy.go

```diff
@@ -45,14 +46,18 @@ func (t TimeOfDay) String() string {
 
 // SortAndDedupeTimesOfDay sorts the slice of times of day and removes duplicates.
 func SortAndDedupeTimesOfDay(tod []TimeOfDay) []TimeOfDay {
-	sort.Slice(tod, func(i, j int) bool {
-		if a, b := tod[i].Hour, tod[j].Hour; a != b {
-			return a < b
+	slices.SortFunc(tod, func(a, b TimeOfDay) int {
+		if n := cmp.Compare(a.Hour, b.Hour); n != 0 {
+
```

Comment:
nit: the linter is complaining about this extra line.

```suggestion
```

I used `gofumpt -d  snapshot/policy/scheduling_policy.go` to find out.

```diff
diff snapshot/policy/scheduling_policy.go.orig snapshot/policy/scheduling_policy.go
--- snapshot/policy/scheduling_policy.go.orig
+++ snapshot/policy/scheduling_policy.go
@@ -48,7 +48,6 @@
 func SortAndDedupeTimesOfDay(tod []TimeOfDay) []TimeOfDay {
 	slices.SortFunc(tod, func(a, b TimeOfDay) int {
 		if n := cmp.Compare(a.Hour, b.Hour); n != 0 {
-
 			return n
 		}
```

Reply:
Fixed with latest commit